test/unit/org/apache/cassandra/service/QueryInfoTrackerTest.java

```diff
@@ -112,6 +112,56 @@ public void testSimpleQueryTracing()
         assertEquals(0, tracker.loggedWrites.get());
     }
 
+    @Test
+    public void testReadQueryTracingWithStaticRows()
+    {
+        int keys = 4;
+        int clustering = 3;
+        String table = KEYSPACE + ".qit_read_static";
+        session.execute("CREATE TABLE " + table + "(k int, c int, v int, sv int static, PRIMARY KEY (k, c))");
```

Comment:
We don't see to have any tests with no clustering columns.  I would add a test for a table with no clustering columns.

And then to cover all bases for static probably another test which only has clustering columns and static columns, not regular columns.

Reply:
This I'd like to do in a separate PR if you don't mind.
I'd like to submit cndb-7393 PR first